index.html

```diff
@@ -436,6 +437,10 @@ <h2>
         circumstances under which a given wake lock may be released by the
         <a>user agent</a>.
       </aside>
+      <p>
+        The <dfn>released</dfn> attribute indicates whether the
+        {{WakeLockSentinel}} has been released. Its initial value is `false`.
```

Comment:
also it will never be true again right?

Reply:
Maybe worth pointing out? Add example?

Reply:
> also it will never be true again right?

It's the other way around: it starts as `false`, but once it becomes `true` it will never become `false` again.

Reply:
right, that is what I meant :)

Reply:
I've tried to address both suggestions, please take a look again.

Reply:
I don't think it's worth changing, but I'd usually put that info in a note, since it's more like a code comment.